models/stg_iterable__campaign_history.sql

```diff
@@ -0,0 +1,48 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_iterable__campaign_history_tmp') }}
+
+),
+
+fields as (
+
+    select
+        /*
+        The below macro is used to generate the correct SQL for package staging models. It takes a list of columns 
+        that are expected/needed (staging_columns from dbt_iterable_source/models/tmp/) and compares it with columns 
+        in the source (source_columns from dbt_iterable_source/macros/).
+        For more information refer to our dbt_fivetran_utils documentation (https://github.com/fivetran/dbt_fivetran_utils.git).
+        */
```

Comment:
this is great! i often wonder how confusing us using this macro is for users lol